data.js

```diff
@@ -0,0 +1,70 @@
+var CarLot = (function(carlot) {
+  // Defining private varible to store parsed JSON data
+  var data = {}
+  // Function to get JSON data and then call a function to build the HTML page
+  carlot.loadInventory = (function() {
```

Comment:
Kudos for the IIFE on your public function here as it works as intended. However, what we are looking for here is the use of a callback to achieve the same results. Take a look at the instructions. The quiz.js file holds the populatePage function (yours is named buildHTML). See if you can get it to work in that context.